{"event": "verbose", "uuid": "bf8c9424-d521-4081-96ed-16634a65e81d", "counter": 7, "stdout": "\u001b[1;35mpackages/ansible/plugins/connection/vmware_tools.py) as it seems to be invalid:\u001b[0m", "start_line": 6, "end_line": 7, "runner_ident": "d7ea8ad8-66f4-4a4e-8796-5321b6351329", "created": "2023-05-16T05:52:39.469283"}